scripts/scriptutils/part_retrieval.py

```diff
@@ -349,26 +354,31 @@ def package_parts_inventory(package: str, targets: List[str] = None) -> PackageI
     inventory = PackageInventory()
 
     # import FASTAs and GenBank
-    for file in sorted(itertools.chain(*(glob.glob(os.path.join(package, f'*{ext}')) for ext in GENETIC_DESIGN_FILE_TYPES['FASTA']))):
+    for file in sorted(itertools.chain(
+            *(glob.glob(os.path.join(package, f'*{ext}')) for ext in GENETIC_DESIGN_FILE_TYPES['FASTA']))):
         is_igem_cache = os.path.basename(file) == IGEM_FASTA_CACHE_FILE
         prefix = iGEM_SOURCE_PREFIX if is_igem_cache else package_stem(package)
         with open(file) as f:
             import_file = ImportFile(file, file_type='FASTA', namespace=prefix)
             for record in SeqIO.parse(f, "fasta"):
                 identity = id_map[record.id] if record.id in id_map else accession_to_sbol_uri(record.id, prefix)
+                print(f'Fasta Part identity is {identity}')
                 inventory.add(import_file, identity)
 
-    for file in sorted(itertools.chain(*(glob.glob(os.path.join(package, f'*{ext}')) for ext in GENETIC_DESIGN_FILE_TYPES['GenBank']))):
+    for file in sorted(itertools.chain(
+            *(glob.glob(os.path.join(package, f'*{ext}')) for ext in GENETIC_DESIGN_FILE_TYPES['GenBank']))):
         is_ncbi_cache = os.path.basename(file) == NCBI_GENBANK_CACHE_FILE
         prefix = NCBI_PREFIX if is_ncbi_cache else package_stem(package)
         with open(file) as f:
             import_file = ImportFile(file, file_type='GenBank', namespace=prefix)
             for record in SeqIO.parse(f, "gb"):
                 if record.name in id_map:
                     identity = id_map[record.name]
+                    # print(f'GB Part identity is {identity}')
```

Comment:
Please remove commented out print statements, here and below.

Reply:
You are right, if prints are needed they can be done locally.